### ISSUE TITLE
✨ Develop the Logo Carousel Component

### ISSUE THREAD

**Issue by bettybondoc** [SSWConsulting/SSW.Website#3318]
cc @wicksipedia @isaaclombardssw @UlyssesMaclaren @adamcogan @miichaelsmedley @louisroa8189

**Description**
Develop the Logo Carousel component for showcasing logos or images on the Consulting Pages. The carousel should support smooth auto-scrolling with hover-to-pause functionality. Customisation options should allow users to tailor the carousel’s appearance and behavior, ensuring flexibility while maintaining a clean and intuitive user interface.

**Acceptance Criteria:**
- The Logo Carousel component includes the following customization options:

> - Background Selector (Transparent, Grey-700, Gradient 1/2/3/4)
> - Heading Text Field
> - Image File Uploader (Repeatable):
>   - Alt Text
>   - Link URL

- The carousel moves automatically by default and stops when hovered over.
- Fully responsive across desktop, tablet, and mobile.
- Works seamlessly on all major browsers (Chrome, Firefox, Safari, Edge).
- Fields can be enabled or disabled for flexibility.
- Code is ready for theming (e.g., Light/Dark modes, custom colours – [see linked PBI](https://github.com/SSWConsulting/SSW.Website/issues/3300)).
- Tested and approved by Wicksy, Louis, and Betty.
- Accessibility standards are met (e.g., WCAG 2.1 AA compliance, including mandatory Alt Text for images).
- Documentation is provided with examples for common use cases (e.g., Logo Showcase, Image Carousel).

**Tasks**

![Image](https://github.com/user-attachments/assets/9b4a74e6-e407-49f3-ab61-8ea60fb8dcfc)
**Figure: Logo Carousel - Desktop**

![Image](https://github.com/user-attachments/assets/54edf1bb-0540-4333-82ea-184ef0a20b26)
**FigureL Logo Carousel - Mobile**

**Comment by isaaclombardssw** [comment]
Mostly done, waiting merge for the Image Text block before I can finish this off

**Comment by isaaclombardssw** [comment]
Merged the image text block stuff to finalise this, but merge into main is blocked by the icon optimisation stuff...

See proposed changes at #3368.

**Comment by Calinator444** [comment]
See status update

https://github.com/SSWConsulting/SSW.Website/issues/3316#issuecomment-2518681507

**Comment by isaaclombardssw** [comment]
See our done video – https://www.youtube.com/watch?v=YWSE8qX30Vw
